templates/directus/index.ts

```diff
@@ -59,24 +61,58 @@ export function generate(input: Input): Output {
     },
   });
 
-  if (input.databaseType === "postgres") {
+  if (input.databaseType === "postgis" || input.databaseType === "mysql") {
+    const appEnv = [];
+    if (input.databaseType === "postgis") {
+      appEnv.push(
+        `POSTGRES_USER=${input.databaseType}`,
+        `POSTGRES_PASSWORD=${databasePassword}`,
+        `POSTGRES_DB=$(PROJECT_NAME)`
+      );
+    } else {
+      appEnv.push(
+        `MYSQL_ROOT_PASSWORD=${randomPassword()}`,
+        `MYSQL_USER=${input.databaseType}`,
+        `MYSQL_PASSWORD=${databasePassword}`,
+        `MYSQL_DATABASE=$(PROJECT_NAME)`
+      );
+    }
     services.push({
-      type: "postgres",
+      type: "app",
       data: {
         projectName: input.projectName,
         serviceName: input.databaseServiceName,
-        password: databasePassword,
+        source: {
+          type: "image",
+          image:
+            input.databaseType === "postgis"
+              ? "postgis/postgis:15-master"
+              : "mysql:8",
+        },
+        env: appEnv.join("\n"),
+        mounts: [
+          {
+            type: "volume",
+            name: "data",
+            mountPath: `/var/lib/${
+              input.databaseType === "postgis" ? "postgresql/data" : "mysql"
+            }`,
+          },
+        ],
+        deploy: {
+          command:
+            input.databaseType === "postgis"
+              ? ""
+              : "docker-entrypoint.sh --default-authentication-plugin=mysql_native_password",
```

Comment:
db commands will be added soon, and this will be possible with native mysql service

Reply:
If you want, let's hold the update or I can release 1 more update at that time. The decision is yours. @deiucanta

Reply:
@deiucanta I have released the update about the new feature.